### PR TITLE
rework devon4j-app tutorial

### PR DESCRIPTION
**Pull request by suvmanda**
rework devon4j-app tutorial

### CODE REVIEW

File: devon4j-app/index.asciidoc [hunk 1]
```diff
@@ -61,12 +69,12 @@ Each component is divided into following layers:
...
-* [data-access layer](https://github.com/devonfw/devon4j/blob/master/documentation/guide-dataaccess-layer.asciidoc) for the data access (esp. persistence).For example, in sampleapp we generated com.example.application.sampleapp.<compoenntname>.dataaccess will contain entity, repositories etc.
+* [data-access layer](https://github.com/devonfw/devon4j/blob/master/documentation/guide-dataaccess-layer.asciidoc) for the data access (esp. persistence).For example, in sampleapp we generated com.example.application.sampleapp.<componentname>.dataaccess will contain entity, repositories etc.
 ====
-====
-### Build devon4j sample application
 
-Build the java project
+
+====
+Now. you have to build a devon4j sample application i.e. sampleapp.
```
Comment (GuentherJulian) [inline devon4j-app/index.asciidoc:77]: Remove the dot after Now

File: devon4j-app/index.asciidoc [hunk 2]
```diff
@@ -42,8 +47,10 @@ sampleapp contains 3 modules i.e api, core and server.
```
Comment (GuentherJulian) [inline devon4j-app/index.asciidoc:47]: Typo in batch section: mave**n**

File: devon4j-app/index.asciidoc [hunk 3]
```diff
@@ -53,6 +60,7 @@ displayContent("devon4j multilayer architecture", [{ "image": "images/devon4j_pr
 As shown in above image,devon4j application follows [multilayered architecture](https://en.wikipedia.org/wiki/Multitier_architecture).
```
Comment (GuentherJulian) [inline devon4j-app/index.asciidoc:60]: Add space before devon4j: image, devon4j

File: devon4j-app/index.asciidoc [hunk 4]
```diff
@@ -76,14 +84,23 @@ buildJava("sampleapp", false)
...
 ====
+
+
 ====
 ## Conclusion
 
 In this tutorial you have learnt how to create devon4j application using command line or via Eclipse. Next you can check for other tutorials such as:
+
  * Java backend code generation using CobiGen (Code Generator)
+
```
Comment (GuentherJulian) [inline devon4j-app/index.asciidoc:95]: When the problem with CobiGen is fixed and this tutorial is working again, then we can link it here. So nothing to do at the moment, but keep in mind to link other existiing tutorials from here

**Review by GuentherJulian — CHANGES_REQUESTED** [review]

File: devon4j-app/index.asciidoc (hunk 4 above)
Comment (dixyushi) [reply devon4j-app/index.asciidoc:93]: As of now I have removed cobigen link and modified the document.

File: devon4j-app/index.asciidoc (hunk 3 above)
Comment (dixyushi) [reply devon4j-app/index.asciidoc]: changed.

File: devon4j-app/index.asciidoc (hunk 2 above)
Comment (dixyushi) [reply devon4j-app/index.asciidoc:47]: corrected.

File: devon4j-app/index.asciidoc (hunk 1 above)
Comment (dixyushi) [reply devon4j-app/index.asciidoc]: Done

**Review by GuentherJulian — APPROVED** [review]

**Pull request merged**